Touch Bar Simulator/TouchBarWindow.swift

```diff
@@ -9,6 +89,43 @@ final class TouchBarWindow: NSPanel {
 		return false
 	}
 
+	private var defaultsObservations: [DefaultsObservation] = []
+
+	func setUp() {
+		let view = self.contentView!
+		view.wantsLayer = true
+		view.layer?.backgroundColor = NSColor.black.cgColor
+
+		let touchBarView = TouchBarView()
+		self.setContentSize(touchBarView.bounds.adding(padding: 5).size)
+		touchBarView.frame = touchBarView.frame.centered(in: view.bounds)
+		view.addSubview(touchBarView)
+
+		defaultsObservations.append(defaults.observe(.windowTransparency) { change in
+			self.alphaValue = CGFloat(change.newValue)
+		})
+		defaultsObservations.append(defaults.observe(.windowDocking) { change in
+			self.docking = change.newValue
+		})
+		defaultsObservations.append(defaults.observe(.showOnAllDesktops) { change in
+			self.showOnAllDesktops = change.newValue
+		})
```

Comment:
Just a thought. Maybe we could add some binding here too?

```swift
defaults.bind(.showOnAllDesktops, to: \.showOnAllDesktop)
```

Where `to` could be a KeyPath.